AsyncAwait.Task2.CodeReviewChallenge/Controllers/HomeController.cs

```diff
@@ -20,20 +20,20 @@ public HomeController(IAssistant assistant, IPrivacyDataService privacyDataServi
         _privacyDataService = privacyDataService ?? throw new ArgumentNullException(nameof(privacyDataService));
     }
 
-    public ActionResult Index()
+    public IActionResult Index()
     {
         return View();
     }
 
-    public ActionResult Privacy()
+    public async Task<IActionResult> Privacy()
     {
-        ViewBag.Message = _privacyDataService.GetPrivacyDataAsync().Result;
+        ViewBag.Message = await _privacyDataService.GetPrivacyDataAsync();
```

Comment:
using await instead of synchronously waiting for result to return the control to calling thread.